services/manifest-repo-export-service/pkg/repository/transformer.go

```diff
@@ -242,29 +235,26 @@ func (c *DeployApplicationVersion) Transform(
 	t TransformerContext,
 	transaction *sql.Tx,
 ) (string, error) {
-	fs := state.Filesystem
+	fsys := state.Filesystem
```

Comment:
Just curious: why are we using `fsys` instead of `fs` all of the sudden? 😄

Reply:
because fs is the same as the name of the import, so goland marks it as a warning.